From ab8cd23b4bb65f0b8a3731dbb3389ecc8620c353 Mon Sep 17 00:00:00 2001
From: steadfasterX <steadfasterX@gmail.com>
Date: Wed, 25 Apr 2018 15:56:18 +0200
Subject: [PATCH 3/5] cryptfs_hw: add missing logging tag

Change-Id: Iba69c6feb976cab2e466d30241f899540eff4b51

cryptfs_hw: Properly define log tag

 * LOG_TAG should be defined before including log.h

vendor/qcom/opensource/cryptfs_hw/cryptfs_hw.c:45:9: warning: 'LOG_TAG'
macro redefined [-Wmacro-redefined]
        ^
system/core/liblog/include/log/log.h:51:9: note: previous definition is
here

Change-Id: Id6fdf6f1582360514de9140bd42feaa698527557
---
 cryptfs_hw.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/cryptfs_hw.c b/cryptfs_hw.c
index 6a3a5b1..2545d67 100644
--- a/cryptfs_hw.c
+++ b/cryptfs_hw.c
@@ -26,6 +26,8 @@
  * IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#define LOG_TAG "Cryptfs_hw"
+
 #include <stdlib.h>
 #include <string.h>
 #include <sys/limits.h>
-- 
2.17.1

